sda/internal/userauth/userauth.go

```diff
@@ -8,14 +8,27 @@ import (
 	"net/url"
 	"os"
 	"path/filepath"
+	"slices"
 	"strings"
 
 	"github.com/lestrrat-go/jwx/v2/jwk"
 	"github.com/lestrrat-go/jwx/v2/jws"
 	"github.com/lestrrat-go/jwx/v2/jwt"
+	"github.com/neicnordic/sensitive-data-archive/internal/config"
 	log "github.com/sirupsen/logrus"
 )
 
+var configAuth *config.Config
+
+func init() {
+	// initialize config
+	var err error
+	configAuth, err = config.NewConfig("auth")
+	if err != nil {
+		log.Debugf("failed to get auth config, reason: %v", err)
+	}
+}
+
```

Comment:
```suggestion
```

This is an internal component used by several applications, as such distinct application config should no take place here.